src/annotator/annotation-sync.js

```diff
@@ -0,0 +1,176 @@
+'use strict';
+
+// AnnotationSync listens for messages from the sidebar app indicating that
+// annotations have been added or removed and relays them to Annotator.
+//
+// It also listens for events from Annotator when new annotations are created or
+// annotations successfully anchor and relays these to the sidebar app.
+function AnnotationSync(bridge, options) {
+  var event;
+  var func;
+  var handler;
+  var method;
```

Comment:
These four variables can be removed by a follow-up PR that refactors the `bind()` usages below.